aws/rust-runtime/aws-sigv4/Cargo.toml

```diff
@@ -27,7 +27,7 @@ hmac = "0.12"
 http = { version = "0.2", optional = true }
 num-bigint = { version = "0.4", optional = true }
 once_cell = "1.8"
-p256 = { version = "0.13.2", features = ["ecdsa"], optional = true }
```

Comment:
intentional downgrade?

Reply:
Yeah, b/c it would otherwise conflict with the version of `digest` required by `hmac`